packages/manager/modules/pci/src/projects/project/kubernetes/kubernetes.controller.js

```diff
@@ -22,10 +23,14 @@ export default class {
   }
 
   getKubernetes({ id: kubeId }) {
-    return this.OvhApiCloudProjectKube.v6().get({
-      serviceName: this.projectId,
-      kubeId,
-    }).$promise;
+    return this.OvhApiCloudProjectKube.v6()
+      .get({
+        serviceName: this.projectId,
+        kubeId,
+      })
```

Comment:
**optional**: Maybe replace the ovh-api-services usage by a `$http` (so we slowly get rid of it)

Reply:
its an existing API, we can do it at once.